cli.js

```diff
@@ -14,7 +15,8 @@ const cli = meow(`
 	  211
 `);
 
-const input = cli.input[0];
+const input = cli.input[0] || cli.flags.pretty;
```

Comment:
So basically, this is to make sure you can do `gzip-size --pretty blah.js` or `gzip-size blah.js --pretty`. Basically just convenience for the user. You see this sort of thing in other packages like `npm install ava -g` or `npm install -g ava`